backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -145,80 +233,188 @@ describe('given some notifications', () => {
+                }),
+                expect.objectContaining({
+                  from: {
+                    __typename: 'Post',
+                    content: 'You have been mentioned in a post',
+                  },
+                  read: false,
+                  createdAt: '2019-08-31T17:33:48.651Z',
+                }),
+                {
+                  createdAt: '2020-01-15T16:33:48.651Z',
+                  read: false,
+                  reason: 'filed_report_on_resource',
+                  from: {
+                    __typename: 'Report',
+                    id: 'reportOnUser',
+                    filed: [
+                      {
+                        reasonCategory: 'discrimination_etc',
+                        reasonDescription: 'This user is harassing me with bigoted remarks!',
+                        reportedResource: {
+                          __typename: 'User',
+                          id: 'badWomen',
+                          name: 'Mrs. Badwomen',
+                        },
+                      },
+                    ],
+                  },
+                },
+                {
+                  createdAt: '2020-01-16T10:33:48.651Z',
+                  read: true,
+                  reason: 'filed_report_on_resource',
+                  from: {
+                    __typename: 'Report',
+                    id: 'reportOnPost',
+                    filed: [
+                      {
+                        reasonCategory: 'other',
+                        reasonDescription:
+                          "This shouldn't be shown to anybody else! It's my private thing!",
+                        reportedResource: {
+                          __typename: 'Post',
+                          id: 'p4',
+                          title: 'Bad Post',
+                          content: 'I am bad content !!!',
+                        },
+                      },
+                    ],
+                  },
+                },
+                {
+                  createdAt: '2020-01-14T12:33:48.651Z',
+                  read: false,
+                  reason: 'filed_report_on_resource',
+                  from: {
+                    __typename: 'Report',
+                    id: 'reportOnComment',
+                    filed: [
+                      {
+                        reasonCategory: 'discrimination_etc',
+                        reasonDescription: 'This user is harassing me!',
+                        reportedResource: {
+                          __typename: 'Comment',
+                          id: 'c4',
+                          content: 'I am bad content in a bad comment to a bad post !!!',
+                        },
+                      },
+                    ],
+                  },
+                },
+              ]),
             },
             errors: undefined,
-          })
+          }
+
+          const response = await query({ query: notificationQuery, variables })
```

Comment:
```suggestion
          const { data: { notifications } } = await query({ query: notificationQuery, variables })
```

Reply:
I don’t like to do this, because I got to the –  in my eyes – useful habit to have the `errors: undefined,` within the object that I compare.
Then the next coder can explicitly see that it’s in the response and not ending up with the same problem I had: There is an undefined error. What I did wrong.
And spend a lot of time to try to find out …